app/templates/learn/index.hbs

```diff
@@ -1,36 +1,59 @@
 <section aria-labelledby="learningember">
   <h1 id="learningember">Learning Ember.js</h1>
   <p>
-    <img src="/images/tomsters/teaching-reverse.png" alt="" role="presentation" class="pull-right img-mascot">
+    <img
+      src="/images/tomsters/teaching-reverse.png"
+      alt=""
+      role="presentation"
+      class="pull-right img-mascot"
+    >
     Are you getting started on your Ember.js adventure? Check out our official reading material:
-  <ul>
-    <li>
-      <a href="https://guides.emberjs.com/current/getting-started/quick-start/"><strong>Quick start</strong><i class="icon-link-ext" aria-hidden="true"></i></a>
-      : An easy breezy introduction to the framework.</li>
-    <li> <a href="https://guides.emberjs.com/current/tutorial/ember-cli/"><strong>Tutorial</strong><i class="icon-link-ext" aria-hidden="true"></i></a>
-    : Follow our tutorial to build, and deploy, your first Ember.js application!</li>
-    <li> <a href="https://guides.emberjs.com/"><strong>Guides</strong><i class="icon-link-ext" aria-hidden="true"></i></a>: Whether you're just getting started and want to get familiar with Ember, or you're looking to refresh your knowledge on a certain feature, the Guides are the place for you.</li>
-  </ul>
+    <ul>
+      <li>
+        <a href="https://guides.emberjs.com/current/getting-started/quick-start/">
+          <strong>Quick start</strong><i class="icon-link-ext" aria-hidden="true"></i>
+        </a>
+        : An easy breezy introduction to the framework.
+      </li>
+      <li>
+        <a href="https://guides.emberjs.com/current/tutorial/ember-cli/">
+          <strong>Tutorial</strong><i class="icon-link-ext" aria-hidden="true"></i>
+        </a>
+        : Follow our tutorial to build, and deploy, your first Ember.js application!
+      </li>
+      <li>
+        <a href="https://guides.emberjs.com/">
+          <strong>Guides</strong><i class="icon-link-ext" aria-hidden="true"></i>
+        </a>
+          : Whether you're just getting started and want to get familiar with Ember, or you're looking to refresh your knowledge on a certain feature, the Guides are the place for you.
```

Comment:
This is adding a space: 
![screen shot 2018-10-26 at 09 15 39](https://user-images.githubusercontent.com/594890/47553743-bd5c4100-d8ff-11e8-93a4-6043f061b280.png)